read_motif_v6.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Nov  2 01:28:58 2020
+
+@author: YB Moon (ymoon06)
+"""
+# assume motif <read
+# use_N is for further implement as a parameter
+def find_overlaps(motif, read, min_overlap, use_n=False):
```

Comment:
Can you please add an option to report only matches that are fully contained in the read?